Backend-Roadmap/Context/AppDbContext.cs

```diff
@@ -22,9 +22,12 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             modelBuilder.Entity<Users>().ToTable("users");
             modelBuilder.Entity<Project>().ToTable("project");
             modelBuilder.Entity<Task>().ToTable("task");
+            modelBuilder.Entity<Epyc>().ToTable("epyc");
```

Comment:
Epic

Reply:
Also assumed it was Apic , but on different versions of the roadmap documentation to follow had this schema. 

![image_2024-05-17_115036552](https://github.com/zesilvapro/RoadmapEQS/assets/151776359/cc6574ef-91c3-4ea0-8088-50a6ee941862)

Reply:
Who made the documentation :smile:

Reply:
As requested per @tentone

Reply:
It is Epic, he didn't notice the error in the document